<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="jenkins-agent-10-25-213-156" failures="0" tests="6" name="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" time="60.528" errors="0" timestamp="2024-02-01T07:58:52 UTC" skipped="0">
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_504_And_No_MaxRetryTime" time="0.068"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_502" time="13.074"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_504" time="2.131"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_429" time="5.100"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_429_And_No_MaxRetryTime" time="0.078"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_503" time="40.077"/>
  <system-out/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest -->
